portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -69,12 +69,62 @@ definition {
 	}
 
 	@description = "Verify that a Form can be submitted with the Paragraph Field"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeSubmitted {
 		property portal.acceptance = "false";
```

Comment:
Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.

Reply:
Yes, I just submitted this change along with the gotoAddField change